mmv1/products/gemini/ReleaseChannelSetting.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: ReleaseChannelSetting
+description: The resource for managing ReleaseChannel settings for Admin Control.
+base_url: projects/{{project}}/locations/{{location}}/releaseChannelSettings
+update_mask: true
+self_link: projects/{{project}}/locations/{{location}}/releaseChannelSettings/{{release_channel_setting_id}}
+create_url: projects/{{project}}/locations/{{location}}/releaseChannelSettings?releaseChannelSettingId={{release_channel_setting_id}}
+update_verb: PATCH
+id_format: projects/{{project}}/locations/{{location}}/releaseChannelSettings/{{release_channel_setting_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/releaseChannelSettings/{{release_channel_setting_id}}
+mutex: projects/{{project}}/locations/{{location}}/releaseChannelSettings/{{release_channel_setting_id}}
+examples:
+  - name: gemini_release_channel_setting_basic
+    min_version: 'beta'
+    primary_resource_id: example
+    vars:
+      release_channel_setting_id: ls1-tf
+      project_id: "aandrei-test"
+autogen_async: false
+autogen_status: UmVsZWFzZUNoYW5uZWxTZXR0aW5n
+parameters:
+  - name: location
+    type: String
+    description: Resource ID segment making up resource `name`. It identifies the resource within its parent collection as described in https://google.aip.dev/122.
+    immutable: true
+    url_param_only: true
+    required: true
+  - name: releaseChannelSettingId
+    type: String
+    description: |-
+      Required. Id of the requesting object.
+      If auto-generating Id server-side, remove this field and
+      release_channel_setting_id from the method_signature of Create RPC
+    immutable: true
+    url_param_only: true
+    required: true
+properties:
+  - name: createTime
+    type: String
+    description: Output only. [Output only] Create time stamp.
+    output: true
+  - name: updateTime
+    type: String
+    description: Output only. [Output only] Update time stamp.
+    output: true
+  - name: labels
+    type: KeyValueLabels
+    description: Optional. Labels as key value pairs.
+  - name: releaseChannel
+    type: String
+    description: |-
+      Optional. Release channel to be used.
+      Possible values:
+      CHANNEL_TYPE_UNSPECIFIED
```

Comment:
While this exists in the proto the _UNSPECIFIED value is almost never valid, so we should remove it from the docs here